core/drivers/stm32_cpu_opp.c

```diff
@@ -75,6 +75,63 @@ static TEE_Result set_opp_voltage(unsigned int opp)
 	return regulator_set_voltage(cpu_opp.regul, cpu_opp.dvfs[opp].volt_uv);
 }
 
+/*
+ * This function returns true if the given OPP voltage can be managed.
+ * If the exact voltage value is not supported by the regulator,
+ * the function may adjust the input parameter volt_uv to a higher
+ * supported value and still return true.
+ */
+static bool opp_voltage_is_supported(struct regulator *regul, uint32_t *volt_uv)
+{
+	const int target_volt_uv = *volt_uv;
```

Comment:
I think their should be a cast here to convert the unsigned value into a signed value.